lib/peep.rb

```diff
@@ -0,0 +1,48 @@
+require 'date'
+require_relative 'peep_factory'
+
+class Peep
+  FILTER_QUERY = 
+  "SELECT * FROM peeps WHERE UPPER(message) LIKE $1
+  ORDER BY date DESC, id DESC;".freeze
+  INSERT_NO_DATE_QUERY = 
+  "INSERT INTO peeps(message) VALUES($1) RETURNING id, message, date".freeze
+  INSERT_QUERY = 
+  "INSERT INTO peeps(message, date) VALUES($1, $2) RETURNING id, message, date".freeze
+  ALL_PEEPS_QUERY =
+  "SELECT * FROM peeps ORDER BY date DESC, id DESC;".freeze
```

Comment:
I question the need for these class constants, given they are only used once each? Makes the query a little neater however I agree.